store/types/store.go

```diff
@@ -38,6 +38,45 @@ type Queryable interface {
 //----------------------------------------
 // MultiStore
 
+type StoreUpgrades struct {
+	New     []string
```

Comment:
`New` isn't really used is it?

Reply:
It is not. I left it as you had it. But then I realized the code auto-creates new stores on mount. I will remove